# Demultiplexing-project

|               | Input         | Simulated data |  pros         | Cons      |
| ------------- | ------------- |------------- | -------------| -------------|
| Vireo  | cell data: thee supported formats <br> 1. cellSNP output folder containing VCF for variant info and sparse matrices AD and DP <br> 2. Vatrix outputs with with three or four files: alt.mtx,  ref.mtx, barcodes.tsv [,SNP.vcf.gz] <br> 3.standard VCF files with variants by cells <br> optional: <br> donor genotype file in VCF format  | https://github.com/single-cell-genetics/vireo/tree/master/simulate | •Can handle partial reference genotype information if genotype is available for a subset of samples| •Does not automatically detect number of individuals in the pool <br> •Does not detect unique SNPs per individuals <br> •Limited to 12 samples <br> •Samples represented by fewer than 100 cells could be missed <br> •Inferred samples cannot be directly identified or linked to metadata, but the estimated genotype states can be used to link the samples to other omics data which can be applied to any read-based assay which provides genotypes|
| scSplit  | •mixed samples BAM file <br> •barcodes whitelist  | *To test the consistency of the model, and the performance of our demultiplexing tool, reference/alternative count matrices were simulated from a randomly selected scRNA-seq BAM file from Zheng et al. [2] and a 32-sample VCF file used in Fig. 2 supplementary data of Kang et al. [6].* <br> Zheng GXY, Terry JM, Belgrader P, Ryvkin P, Bent ZW, Wilson R, et al.Massively parallel digital transcriptional profiling of single cells. Nat Commun. 2017; 8. https://doi.org/10.1038/ncomms14049. <br> http://support.10xgenomics.com/single-cell/datasets <br> Kang HM, Subramaniam M, Targ S, Nguyen M, Maliskova L, McCarthy E, et al.Multiplexed droplet single-cell RNA-sequencing using natural genetic variation. Nat Biotechnol. 2017; 36:89. <br> https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96583 | •does not require genotype data nor a list of common variants <br> •retains accuracy under shallow read depth (tested at 2800 reads per cell) <br> •detects unique SNPs per individuals| • has been tested on only 8 real mixed samples <br> •cannot take reference SNP genotype|
| Souporcell  | •possorted_genome_bam.bam from cellranger <br> •barcodes.tsv from cell ranger <br> optional: <br> •reference FASTA file  | *To generate realistic data with known ground truth, we sequenced five lines of induced pluripotent stem cells (iPSCs) from the Human iPSC Initiative17 with the 10x Chromium single-cell system, both individually and in a mixture of all five lines (with three replicates of the mixture). Each mixture contained 5,000–7,000 cells and ~25,000 UMIs per cell (Supplementary Table 1). We first synthetically mixed 20% of the cells from the five individual samples while retaining their sample of origin. To make the synthetic mixture as close to real data as possible, we also simulated 6% doublets by switching all of the reads’ barcodes from one cell to that of another cell and 5% ambient RNA by randomly switching cell barcodes for 5% of the reads.* <br> https://www.ebi.ac.uk/ena/browser/view/ERS2630499 <br> https://www.ebi.ac.uk/ena/browser/view/ERS2630500 <br> https://www.ebi.ac.uk/ena/browser/view/ERS2630501 | •reference genotype is optional <br> •identifies proportion of ambient RNA <br> •can identify overlapping clusters from different datasets allows genotype information to be extracted| •cannot automatically detect number of individuals in the pool <br> •cannot detect unique SNPs per individuals <br> •Low signal-to-noise ratio due to decreased numbers of UMIs per cell and high numbers of donors, causing local maxima|
| Demuxalot  | •genotypes.vcf <br> •barcodes.csv <br> •sorted_alignment.bam|Used the PBMC dataset from demuxlet paper <br> https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96583| |•Data efficiency: Aims to maximize genotype information with as few additional putative SNVs as needed. Data efficiency allows for scalabilty to more dornors and better exploitation of deep sequencing datasets |•none mentioned| 
| Demuxlet  | •SAM/BAM/CRAM produced by any barcoded single cell RNA-seq <br> •VCF/BCF file containing the genotype, posterior probability, or genotype likelihood  | *For a pool of eight individuals and a set of uncorrelated SNPs, each with 50% minor allele frequency (MAF), four reads overlapping SNPs are sufficient to uniquely assign a singlet to the donor of origin (Fig. 1b) and 20 reads overlapping SNPs can distinguish every sample with >98% probability in simulation.* <br> https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96583 | •gold standard | •reference genotyoe is necessary <br> •does not automatically detect number of individuals in the pool
•does not detect unique SNPs per individual|
